operator/config/manager/manager.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
 spec:
   selector:
     matchLabels:
-      control-plane: controller-manager
+      app.kubernetes.io/component: module-manager.kyma-project.io
```

Comment:
This will break if you do not change the labels in the template of the deployment

Reply:
You mean break if people directly apply this manager.yaml? In this case, I think we should avoid to do this, the final resources should be generated always by kustomize build command.

Reply:
This will be added automatically, so you can remove this label